components/cards/InspectionPersonsNumberCard.vue

```diff
@@ -32,15 +32,16 @@
 </template>
 
 <script>
-import Data from '@/data/data.json'
-import formatGraph from '@/utils/formatGraph'
+import formatGraph from '@/libraries/utils/formatGraph'
 import TimeBarChart from '@/components/TimeBarChart.vue'
+import { Registry } from '@/libraries/Registry'
 
 export default {
   components: {
     TimeBarChart
   },
   data() {
+    const Data = Registry.DataRepository.data
```

Comment:
```suggestion
    const { inspection_persons } =  Registry.DataRepository.data
```
とすると．この後の行がすっきり書けると思います．
他の component との統一感を大事にするならいまのままでもいいと思いますが．

Reply:
https://github.com/tokyo-metropolitan-gov/covid19/pull/3008#issuecomment-611329829